de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/table/api/impl/ADTableDAO.java

```diff
@@ -246,10 +246,13 @@ public I_AD_Table retrieveTable(final String tableName)
 				.addOnlyActiveRecordsFilter()
 				.addOnlyContextClient()
 				.addEqualsFilter(I_AD_Column.COLUMNNAME_AD_Table_ID, table.getAD_Table_ID())
+				.orderBy()
+				.addColumnAscending(I_AD_Column.COLUMNNAME_Created)
```

Comment:
very small detail, usually we in such cases we are ordering by PK and not by Created